lib/xenops_server.ml

```diff
@@ -1185,8 +1185,11 @@ let rec perform_atomic ~progress_callback ?subtask ?result (op: atomic) (t: Xeno
 			VM_DB.signal id
 		| VM_unpause id ->
 			debug "VM.unpause %s" id;
-			B.VM.unpause t (VM_DB.read_exn id);
-			VM_DB.signal id
+			let power = (B.VM.get_state (VM_DB.read_exn id)).Vm.power_state in
+			begin match power with
+			| Paused -> B.VM.unpause t (VM_DB.read_exn id); VM_DB.signal id
+			| _ -> debug "Cannot perform VM_unpause. VM %s is not paused" id
```

Comment:
Let's use the `info` log level, and make it sound less like an error, e.g. "VM %s is already unpaused."